SPECS/kubevirt/kubevirt.spec

```diff
@@ -204,13 +353,38 @@ install -p -m 0644 cmd/virt-handler/nsswitch.conf %{buildroot}%{_datadir}/kube-v
 %doc README.md
 %{_bindir}/virt-operator
 
+%files pr-helper-conf
+%license LICENSE
+%doc README.md
+%dir %{_datadir}/kube-virt
+%dir %{_datadir}/kube-virt/pr-helper
+%{_datadir}/kube-virt/pr-helper
+
+%files manifests
+%license LICENSE
+%doc README.md
+%dir %{_datadir}/kube-virt
+%dir %{_datadir}/kube-virt/manifests
+%{_datadir}/kube-virt/manifests/release
+
 %files tests
 %license LICENSE
 %doc README.md
 %dir %{_datadir}/kube-virt
+%dir %{_datadir}/kube-virt/manifests
 %{_bindir}/virt-tests
+%{_datadir}/kube-virt/manifests/testing
+
+%files -n obs-service-kubevirt_containers_meta
+%license LICENSE
+%doc README.md
+%dir %{_prefix}/lib/obs
+%{_prefix}/lib/obs/service
 
 %changelog
+* Wed Mar 13 2024 Elaine Zhao <elainezhao@microsoft.com> - 1.2.0-1
+- Bump package version to 1.2.0 from 0.59.0
+
```

Comment:
Can you add the link/URL where you found the requirement to add the `pr-helper-conf`, `manifests` and other new packages along with the host of changes required to the SPEC file.